editor/plugins/mesh_library_editor_plugin.cpp

```diff
@@ -201,7 +201,7 @@ void MeshLibraryEditor::_import_scene_cbk(const String &p_str) {
 	ERR_FAIL_COND(ps.is_null());
 	Node *scene = ps->instance();
 
-	ERR_FAIL_COND(!scene);
+	ERR_FAIL_COND_MSG(!scene, "Cannot instance PackedScene node.");
```

Comment:
Should be "instantiate" (our API is wrong :P). Maybe "Cannot create a PackedScene instance from '" + p_str + "'."